js/views/chatview.js

```diff
@@ -388,18 +386,15 @@
 			return data;
 		},
 
-		_onAddModel: function(model, collection, options) {
-			this.$el.find('.emptycontent').toggleClass('hidden', true);
+		_onAddModelStart: function() {
+			this._virtualList.appendElementStart();
 
-			var $newestComment = this.$container.children('.comment').last();
-			var scrollToNew = $newestComment.length > 0 && this._getCommentTopPosition($newestComment) < this.$container.outerHeight();
+			this._scrollToNew = this._virtualList.getLastElement() === this._virtualList.getLastVisibleElement();
+		},
 
+		_onAddModel: function(model, collection) {
```

Comment:
/drone/src/github.com/nextcloud/spreed/js/views/chatview.js
395:32 error 'collection' is defined but never used no-unused-vars

Reply:
Fixed.